Tutorials/Add-Google-Authentication/index.md

```diff
@@ -57,7 +57,7 @@ Now, fill in your email and leave the page open - you will need the Client ID an
 
 ## Installing Visual Studio packages
 
-Now that you have the Google API all set up you have to go to your existing solution in Visual Studio. If you don't know how to clone down a Cloud site to use in Visual Studio then there is an excellent guide [here!](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Visual-Studio/)
+Now that you have the Google API all set up you have to go to your existing solution in Visual Studio. If you don't know how to clone down a Cloud site to use in Visual Studio then there is an excellent guide [here!](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Working-With-Visual-Studio/)
```

Comment:
Ah, good catch here! 😄 

Shouldn't we instead change the link to `../../Umbraco-Cloud/Set-Up/Working-with-Visual-Studio/` ? 😄 
Since it is an internal link after all!

Reply:
You absolutely right!